VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -29,20 +31,30 @@ final public class AuthenticationController
     let configuration: AppConfiguration
     let client: VimeoClient
     
+    /// We need to use a separate client to make the actual auth requests, to ensure it's unauthenticated
+    private let authenticatorClient: VimeoClient
```

Comment:
Nah, we still configure the client on a successful auth, and we use its configuration